pydantic/dataclasses.py

```diff
@@ -1,23 +1,32 @@
-from typing import TYPE_CHECKING, Any, Callable, Dict, Generator, Optional, Type, TypeVar, Union
+import dataclasses
```

Comment:
I have a suspicion that importing `dataclasses` globally had a big effect on overall import time.

I think easiest to move it back to imports in functions, we could benchmark to see the performance of making this global.

Reply:
Yes I know I remember when you changed some imports to be local in order to improve overall import time.
I forgot to import it back to all functions.